acceptance/acceptance_test.go

```diff
@@ -310,6 +318,12 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	// User replacements come last:
 	repls.Repls = append(repls.Repls, config.Repls...)
 
+	if config.SaveRepls {
+		replsJson, err := json.MarshalIndent(repls.Repls, "", "  ")
+		require.NoError(t, err)
+		testutil.WriteFile(t, filepath.Join(tmpDir, ReplsFile), string(replsJson))
```

Comment:
It might be unlikely to happen but if my test script also produces repls.json this will override it, correct? Shall we error out in this case just to be safe?

Reply:
This is written before your script is run, so your script will override it, which will break repls.json. 

You should be able to modify your script to use different file or rename the files like we do with .gitignore.